compiler/build.gradle

```diff
@@ -75,6 +75,7 @@ model {
   binaries {
     all {
       if (toolChain in Gcc || toolChain in Clang) {
+        cppCompiler.define("GRPC_VERSION", version)
```

Comment:
I don't understand why but if I put it one line above, outside of the `if` to apply it globally, for all tool chains, it fails with:

```* Where:
Build file '/Users/lukasz/Development/grpc-java/compiler/build.gradle' line: 77

* What went wrong:
A problem occurred configuring project ':grpc-compiler'.
> Exception thrown while executing model rule: all { ... } @ compiler/build.gradle line 76, column 5
   > Could not find property 'cppCompiler' on classes 'main'.
```

Hence I duplicated this line for visual cpp, below.